frontend/src/components/patient/SearchPatientForm.js

```diff
@@ -287,7 +308,19 @@ function SearchPatientForm(props) {
                       disallowFutureDate={true}
                     />
                   )}
-                </Field>
+                </Field>{" "}
+                <br />{" "}
+                <Toggle
+                  labelText="crResult"
+                  labelA="false"
+                  labelB="true"
+                  id="toggle-cr"
+                  toggled={isToggled}
+                  onClick={() => {
+                    toggle();
+                    setFieldValue("crResult", !isToggled);
+                  }}
+                />
```

Comment:
you should position this togle button on the same horizontal level as the search buttons